### PR TITLE
tuifimanager: 2.3.4 -> 3.0.0

### PR DESCRIPTION
**Pull request by michaelBelsanti**
###### Description of changes
Updated tuifimanager to v.3.0.0
https://github.com/GiorgosXou/TUIFIManager/releases/tag/v.3.0.0

###### Things done
Switched format, no longer needs checks disabled. Added changelogs.

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by michaelBelsanti** [comment]
Result of `nixpkgs-review pr 235635` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>tuifimanager</li>
    <li>tuifimanager.dist</li>
  </ul>
</details>

**Comment by Stunkymonkey** [comment]
Result of `nixpkgs-review pr 235635` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>tuifimanager</li>
    <li>tuifimanager.dist</li>
  </ul>
</details>

**Review by Stunkymonkey — APPROVED** [review]

**Comment by michaelBelsanti** [comment]
I think it could also be backported, doesn't have any breaking changes.

File: pkgs/applications/file-managers/tuifimanager/default.nix [hunk 1]
```diff
@@ -1,27 +1,35 @@
...
     substituteInPlace pyproject.toml \
       --replace "Send2Trash == 1.8.0" "Send2Trash >= 1.8.0"
   '';
 
-  propagatedBuildInputs = with python3Packages; [ unicurses send2trash ];
-  pythonImportsCheck = [ "TUIFIManager" ];
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+    python3.pkgs.setuptools-scm
+  ];
```
Comment (SuperSandro2000) [inline pkgs/applications/file-managers/tuifimanager/default.nix:26]:
```suggestion
  nativeBuildInputs = with python3.pkgs; [
    setuptools
    setuptools-scm
  ];
```

**Pull request merged**